run-test.sh

```diff
@@ -29,7 +29,7 @@ usage()
     echo "                                      default: <repo_root>/bin/Product/<OS>.x64.<ConfigurationGroup>"
```

Comment:
We're planning on not using this any more.  @iamjasonp -- do you know if it is used in any remaining Jenkins jobs?  To test this will require manually running on a Linux box.  Personally I think it would be safer not to checkin this file unless something is broken.  If we use it, changes could break us.  If we don't use it, we should nuke it.